jaq-std/tests/std.rs

```diff
@@ -127,6 +127,60 @@ yields!(
     false
 );
 
+yields!(logb_nan, "nan | logb | isnan", true);
+yields!(logb_inf, "infinite | logb | . == infinite", true);
+yields!(logb_neg_inf, "-infinite | logb | . == infinite", true);
+yields!(logb_zero, "0 | logb | . == -infinite", true);
+yields!(
+    logb_range,
+    "[-2.2, -2, -1, 1, 2, 2.2] | map(logb)",
+    [1.0, 1.0, 0.0, 0.0, 1.0, 1.0]
+);
+
+yields!(
+    drem_nan,
+    "[drem(nan, 1; nan, 1)] == [nan, nan, nan, 0.0]",
+    true
+);
+yields!(
+    drem_range,
+    "[drem(3.5, -4; 6, 1.75, 2)]",
+    [-2.5, 0.0, -0.5, 2.0, -0.5, -0.0]
+);
+
+yields!(
+    scalb_nan,
+    "[scalb(nan, 1; nan, 1)] == [nan, nan, nan, 2.0]",
+    true
+);
+yields!(
+    scalb_range,
+    "[scalb(-2.5, 0, 2.5; 2, 3)]",
```

Comment:
It would be good to test this filter with float values as second arguments.

Reply:
I agree, though in general I'm wary about including exact matches with approximations to irrational numbers. I guess I could include a test similar to what I did with `sin` and `cos` (which uses the pythagorean identity). Another option is to test for intervals, e.g. I know that `scalb(1; 1) < scalb(1; 1.5) < scalb(1; 2)`.

This is the reason for the next test: I can compare the results with an equivalent implementation using `pow`. Still, I'm not sure about either approach. If you're OK with having exact matches against fully-expanded f64s, then I'll include those cases as well.

Reply:
How about something like `[scalb(-2.5, 0, 2.5; 2, 3) * 1000 | round]`?

Reply:
Brilliant!